test/wasm/rlexe.xml

```diff
@@ -21,15 +21,13 @@
     <compile-flags>-on:wasm</compile-flags>
   </default>
 </test>
-<!--
 <test>
     <default>
        <files>f32.js</files>
        <baseline>f32.baseline</baseline>
        <compile-flags>-on:Wasm</compile-flags>
     </default>
 </test>
--->
```

Comment:
It was disabled for razzle, since I am disabling the whole folder for razzle, I wanted to ake the chance to reenable it before I forget